dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -48,6 +49,7 @@ func init() {
 		fmt.Sprintf("an optional target name for the generated tarball. The default is alluxio-%v.tar.gz. The string %q will be substituted with the built version. "+
 			`Note that trailing ".tar.gz" will be stripped to determine the name for the Root directory of the generated tarball`, versionMarker, versionMarker))
 	cmdSingle.Flags.StringVar(&mvnArgsFlag, "mvn-args", "", `a comma-separated list of additional Maven arguments to build with, e.g. -mvn-args "-Pspark,-Dhadoop.version=2.2.0"`)
+	cmdSingle.Flags.StringVar(&buildUIFlag, "build-ui", "true", `set this flag to false to skip building the webui`)
```

Comment:
I would make this a boolean instead of a string var. Then we can simply call the flag `skip-build-ui` or `skip-ui` and all they have to do is provide it without an argument.

Reply:
Fixed.